src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,23 @@ const createCustomerValidator = () => {
     });
 };
 
+const getCustomersAllValidator = () => {
+    return celebrate({
+        [Segments.QUERY]: Joi.object().keys({
+            offset: Joi.string(),
```

Comment:
O campo `offset` deveria ser um número inteiro, aqui no Joi seria o lugar certo para validar isso e já definir o valor padrão caso o usuário não informasse nada.

```suggestion
            offset: Joi
                .number()
                .integer()
                .min(0)
                .default(0),
```